### DISCUSSION TITLE
RenBTC is stuck

### DISCUSSION THREAD

**Discussion by sylvian3111** [wormhole-foundation/wormhole#349] [Problems / bugs] [answered]
Hello
0xf68f20ad3c46a901e792336c929afe6c6d92b7a5441d5159e3e6111ab9bc0f7e

**Comment by leoluk** [comment, marked as answer]
We're looking into it - looks like the token account was created, but the VAA was never posted: https://explorer.solana.com/address/9WxTDZVRw4y4auZnALcSuTj1zq1bhgXcjP3gHWfpEmVn

**Reply by sylvian3111** [reply]
And what am I supposed to do?

**Reply by leoluk** [reply]
This would've been easier if you reached out immediately - this is almost 40 days ago. I reached out to the guardian set to see if they still have logs, otherwise, we may have to manually replay the block to get the signed VAA.

**Reply by leoluk** [reply]
There you go: https://explorer.solana.com/tx/2kr3oPyY6qmsMKUXV36h2Q9Ht2meVxkFYYHqT3Dd87VBFJcCneHZUtusRyq5k4G65v3FxSB5WW4bnBeSpctokDxm

**Reply by sylvian3111** [reply]
Thank you
